File: Source/CodeTemplate.vb
Imports System
Imports System.IO
Imports System.Threading

Public Class CSharpLab
    Public Shared Sub Test()
        For i As Integer = 0 To 19
            Thread.Sleep(500)
            Console.Write("Hello, World! ")
        Next
        Console.WriteLine()
        Console.WriteLine("Done!")
    End Sub
End Class

File: Source/CodeTemplate.cs
using System;
using System.IO;
using System.Threading;

public class CSharpLab
{
    public static void Test()
    {
        for (int i = 0; i < 20; i++)
        {
            Thread.Sleep(500);
            Console.Write("Hello, World! ");
        }
        Console.WriteLine("\nDone!");
    }
}

